NotebookAnalyzers/utils.py

```diff
@@ -79,8 +146,20 @@ def markdown_titles(notebook):
 
 
 def markdown_distribution(notebook):
-    """The function takes a dict representing a notebook and, dividing the notebook in four sections, 
-    returns the percentage of markdown rows in each section out of the total markdown rows"""
+    """
+        Distribution of markdown rows in the 4 sections of the notebook
+
+        Args:
+            notebook(dic): python dictionary object representing the jupyter notebook
+        Returns:
+            distributions: array of 4 elements, each number representing, for each section, the percentage of markdown rows out of the total rows
+                           distributions[0] = percentage of markdown rows in the first section
+                           distributions[1] = percentage of markdown rows in the second section
```

Comment:
Piuttosto che "section" userei "quarter" o "25%" (e.g., "in the first quarter of the notebook").

"Section" temo che non si capisca, è un po' troppo generico.